src/translator.js

```diff
@@ -109,7 +109,7 @@ Translator.prototype.reloadTranslations = function(type, onComplete) {
           "] (this may take a while)"
       );
       _.each(memberships, function(m) {
-        getMembershipsThrottled(m.idMember, function(err, user) {
+        getMemberships(m.idMember, function(err, user) {
```

Comment:
no I don't think so, I was just testing something and forgot that this PR hadn't been closed. I'll revert the commit.